javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -109,7 +115,9 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
             .ignore(any(), skipClassLoader());
 
     ignoredAgentBuilder =
-        ignoredAgentBuilder.or(globalIgnoresMatcher(skipAdditionalLibraryMatcher));
```

Comment:
`skipAdditionalLibraryMatcher` is now unused?

Reply:
👍 removed